services/updates-svc/stories/updates_test.go

```diff
@@ -0,0 +1,52 @@
+package stories
+
+import (
+	"common/auth"
+	"context"
+	pb "gen/services/updates_svc/v1"
+	"github.com/stretchr/testify/assert"
+	"io"
+	"testing"
+	"time"
+)
+
+func TestOpenAndClosingReceiveUpdatesStream(t *testing.T) {
+	ctx := context.Background()
+	updatesClient := updatesServiceClient()
+
+	req := &pb.ReceiveUpdatesRequest{}
+	stream, err := updatesClient.ReceiveUpdates(ctx, req)
+	assert.NoError(t, err)
+	assert.NoError(t, stream.CloseSend())
+}
+
+func TestAutoClosingWhenTokenExpiresReceiveUpdateStream(t *testing.T) {
+	ctx := context.Background()
+	updatesClient := updatesServiceClient()
+
+	auth.FakeTokenValidFor = time.Second * 3
+	upperBound := auth.FakeTokenValidFor + (time.Second * 1)
```

Comment:
there is time.Round() and time.Truncate()

Reply:
Uhiii very nice!